nengo/spa/thalamus.py

```diff
@@ -188,6 +189,8 @@ def add_route_effect(self,
             subdim = self.subdim_channel
             if isinstance(target_module, nengo.spa.Buffer):
                 subdim = target_module.state.dimensions_per_ensemble
+            elif isinstance(source_module, nengo.spa.Buffer):
+                subdim = source_module.state.dimensions_per_ensemble
```

Comment:
Ah. Righto. But this does mean that if we are using custom modules, this code won't be able to check for them, yes?